sql/plsql/PlSqlParser.g4

```diff
@@ -520,6 +520,7 @@ create_function_body
         | default_collation_clause
         | parallel_enable_clause
         | result_cache_clause
+        | PIPELINED
```

Comment:
Shouldn't it be added to `regular_id` (or `non_reserved_keywords_pre12c`, `non_reserved_keywords_in_12c`, `non_reserved_keywords_in_18c`) instead?

Reply:
You mean to add `PIPELINED` ? It is already in `regular_id` rule. 

I've moved `PIPELINED` that was before `(IS | AS)` into a "subrule with clauses", because in practice `PIPELINED` could be followed by `PARALLEL_ENABLE` in standalone function definition (like in [example from old v9.2 docs](https://docs.oracle.com/cd/B10501_01/appdev.920/a96624/08_subs.htm#20094) or code example I've committed to this PR).